src/crl.rs

```diff
@@ -733,12 +732,88 @@ impl<'a> IssuingDistributionPoint<'a> {
     }
 
     /// Return the distribution point names (if any).
-    #[allow(dead_code)] // TODO(@cpu): remove this once used in CRL validation.
     pub(crate) fn names(&self) -> Result<Option<DistributionPointName<'a>>, Error> {
         self.distribution_point
             .map(|input| DistributionPointName::from_der(&mut untrusted::Reader::new(input)))
             .transpose()
     }
+
+    /// Returns true if the CRL issuing distribution point has a scope that could include the cert
```

Comment:
Can we add a one-line summary at the top?

Reply:
Done.